.travis.yml

```diff
@@ -112,7 +112,7 @@ install:
     - ./scripts/install_deps.sh
 before_script:
     - ./scripts/prepare_ccache.sh
-    - ./scripts/build.sh $TRAVIS_BUILD_TYPE
+    - travis_wait 80 ./scripts/build.sh $TRAVIS_BUILD_TYPE
```

Comment:
Sometimes, the above `brew install llvm` takes more than 10 minutes.  So `travis_wait` should also be there.